### PR TITLE
Adding pneumonia example to repository for blog post purposes

### PR DESCRIPTION
**Pull request by rpytel1**
## Description

This PR adds an example of XAI on pneumonia data, to the repository so that you can better understand how to use our library. Needed for the blog post.

### CODE REVIEW

**Comment by rpytel1** [comment]
I have updated the notebook to work with the latest version of the library. @adamwawrzynski I would appreciate a review soon as I want to update the blogpost with the proper link.

File: example/notebooks/pneumonia.ipynb [hunk 1]
```diff
@@ -0,0 +1,1083 @@
...
+    "Double check if you need to reinstall linux libaries related to OpenCV"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!apt-get update && apt-get install ffmpeg libsm6 libxext6  -y"
```
Comment (adamwawrzynski) [inline example/notebooks/pneumonia.ipynb:25]: I've got this message when executing this cell:
```bash
Reading package lists... Done
E: Could not open lock file [/var/lib/apt/lists/lock](https://file+.vscode-resource.vscode-cdn.net/var/lib/apt/lists/lock) - open (13: Permission denied)
E: Unable to lock directory [/var/lib/apt/lists/](https://file+.vscode-resource.vscode-cdn.net/var/lib/apt/lists/)
W: Problem unlinking the file [/var/cache/apt/pkgcache.bin](https://file+.vscode-resource.vscode-cdn.net/var/cache/apt/pkgcache.bin) - RemoveCaches (13: Permission denied)
W: Problem unlinking the file [/var/cache/apt/srcpkgcache.bin](https://file+.vscode-resource.vscode-cdn.net/var/cache/apt/srcpkgcache.bin) - RemoveCaches (13: Permission denied)
```

File: example/notebooks/pneumonia.ipynb [hunk 2]
```diff
@@ -0,0 +1,1083 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_kg_hide-input": true,
+    "_kg_hide-output": true,
+    "id": "ZghMYdMHMGuj"
+   },
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('./covid-chestxray-dataset/metadata.csv')\n",
```
Comment (adamwawrzynski) [inline example/notebooks/pneumonia.ipynb:117]: Missing file. You should explain how to download/create this file as it is not present in the FoXAI repository. You may add a sentence that user have to manually download the dataset from Kaggle and clone the GitHub repository.

Maybe at the cell above add something like this:
`!git clone https://github.com/ieee8023/covid-chestxray-dataset`

File: example/notebooks/pneumonia.ipynb [hunk 3]
```diff
@@ -0,0 +1,1083 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "yvdZqbdhLQLy"
+   },
+   "outputs": [],
+   "source": [
+    "shutil.rmtree('./COVID19-DATASET/train/covid19')\n",
```
Comment (adamwawrzynski) [inline example/notebooks/pneumonia.ipynb:143]: If there is no such file or directory it raises `FileNotFoundError`. You should check if it exists first.

File: example/notebooks/pneumonia.ipynb [hunk 4]
```diff
@@ -0,0 +1,1083 @@
...
+    "os.makedirs('./COVID19-DATASET/train/covid19')\n",
+    "shutil.rmtree('./COVID19-DATASET/train/normal')\n",
```
Comment (adamwawrzynski) [inline example/notebooks/pneumonia.ipynb:145]: Same here.

File: example/notebooks/pneumonia.ipynb [hunk 5]
```diff
@@ -0,0 +1,1083 @@
...
+    "Installing most needed libaries "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install scikit-learn foxai tqdm"
```
Comment (adamwawrzynski) [inline example/notebooks/pneumonia.ipynb:41]: I've got error with missing `ipywidgets` package in model training function call.

```suggestion
    "!pip install scikit-learn foxai tqdm ipywidgets"
```

File: example/notebooks/pneumonia.ipynb [hunk 6]
```diff
@@ -0,0 +1,1083 @@
...
+    "### Testset Creation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shutil.rmtree('./COVID19-DATASET/test/covid19')\n",
```
Comment (adamwawrzynski) [inline example/notebooks/pneumonia.ipynb:734]: Same here. You have to check if the directory exists.

File: example/notebooks/pneumonia.ipynb [hunk 7]
```diff
@@ -0,0 +1,1083 @@
...
+    "os.makedirs('./COVID19-DATASET/test/covid19')\n",
+    "shutil.rmtree('./COVID19-DATASET/test/normal')\n",
```
Comment (adamwawrzynski) [inline example/notebooks/pneumonia.ipynb:736]: Same here. You have to check if the directory exists.

**Review by adamwawrzynski — CHANGES_REQUESTED** [review]
I would add a more verbose description of what's going in in cells with code. Take a look at this example: https://colab.research.google.com/drive/1WEpVpFSq-oL1Ejugr8Ojb3tcbqXIOPBg.

File: example/notebooks/pneumonia.ipynb (hunk 2 above)
Comment (rpytel1) [reply example/notebooks/pneumonia.ipynb:117]: fixed

File: example/notebooks/pneumonia.ipynb (hunk 3 above)
Comment (rpytel1) [reply example/notebooks/pneumonia.ipynb:143]: fixed

File: example/notebooks/pneumonia.ipynb (hunk 4 above)
Comment (rpytel1) [reply example/notebooks/pneumonia.ipynb:145]: fixed

File: example/notebooks/pneumonia.ipynb (hunk 5 above)
Comment (rpytel1) [reply example/notebooks/pneumonia.ipynb:41]: fixed

File: example/notebooks/pneumonia.ipynb (hunk 6 above)
Comment (rpytel1) [reply example/notebooks/pneumonia.ipynb:734]: fixed

File: example/notebooks/pneumonia.ipynb (hunk 7 above)
Comment (rpytel1) [reply example/notebooks/pneumonia.ipynb:736]: fixed

File: example/notebooks/pneumonia.ipynb (hunk 1 above)
Comment (adamwawrzynski) [reply example/notebooks/pneumonia.ipynb:25]: Agree.

**Comment by rpytel1** [comment]
> I would add a more verbose description of what's going in in cells with code. Take a look at this example: https://colab.research.google.com/drive/1WEpVpFSq-oL1Ejugr8Ojb3tcbqXIOPBg.

I have added more verbosity to the notebook. @adamwawrzynski, if you have not further changes suggestion I would like to merge it.

**Review by adamwawrzynski — APPROVED** [review]
LGTM

**Pull request merged**